### DISCUSSION TITLE
Parsing additional fields (volume, pages, etc) from entry

### DISCUSSION THREAD

**Discussion by sinauq** [emacs-citar/citar-org-roam#20] [Q&A] [answered]
Hi,
is there a way to parse arbitrary fields from a bibtex entry, such as volume, pages and so on?

I tried 
`(setq citar-additional-fields '("volume" "number" "pages" ))`

but it looks like the code doesn't consult this variable that much.

**Comment by bdarcus** [comment, marked as answer]
If you set that variable like you did, the cached entries (accessed by `citar-get-entry`) will include those fields if present.

But yeah, it's not all that useful, since the code will automatically cache whatever fields are specified in the templates.

What are you trying to do?

If it's just to modify the output for a new note, you can modify the `citar-org-roam-note-title-template` template variable.

PS - I opened the discussion feature, and moved the issue there.

**Comment by sinauq** [comment]
Yes, it looks like it is working with `citar-org-roam-note-title-template`. I was trying to do it using the `citar-templates`. Not a issue for me, since I wanted to add these fields to my note template. 

Thanks!
